features/tweet.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Tweet
+
+  Scenario: Tweet Creation
```

Comment:
On the other hand, you might prefer having cleaner scenarios and take care of most stuff in project-specific step definitions. Since Derek found that style guide, I would take a look at that and do whatever makes sense to you.